libmemcached-1.0/storage.h

```diff
@@ -76,6 +85,14 @@ memcached_return_t memcached_cas(memcached_st *ptr,
                                  time_t expiration,
                                  uint32_t flags,
                                  uint64_t cas);
+LIBMEMCACHED_API
+memcached_return_t memcached_cas_vb(memcached_st *ptr,
+                                    const char *key, size_t key_length,
+                                    const uint16_t vbucket,
```

Comment:
vbucket should go on previous line as it's associated with the key (that's what you've done for the other methods)?